Domain Connect Spec Draft.adoc

```diff
@@ -1325,11 +1325,11 @@ indicates that calls to apply a template synchronously will be digitally
 signed. This element contains the domain name for querying the TXT
 record from DNS that contains the public key information.
 
-|*Synchronous Redirect Domain*
+|*Synchronous Redirect Domains*
 |String
 |syncRedirectDomain

```

Comment:
In order to prevent this change from breaking current implementations of this spec, would it make sense to introduce this as a new template param like `syncRedirectDomains`? We can then mark the singular parameter (`syncRedirectDomain`) as deprecated and remove it in a future version of the spec or agreed upon date.

Reply:
+1

Reply:
I see renaming this creating more churn than leaving alone.  Reality is that no template uses this yet, and once people start expecting more than one in here the DNS Providers will only be able to onboard them once they implement the feature.  You rename, you just create more random variables.

Reply:
Sounds good to me. I'm not sure what sort of adoption rate there had been historically and only wanted to ensure this wouldn't cause issues upstream. Thanks @arnoldblinn